### PR TITLE
Fix/post data param

### LINKED ISSUE #28: postData.params with value set to empty String crashes converter
__Steps to reproduce__

Add following to any HAR then run convert
```json
{
    "postData": {
      "mimeType": "application/x-www-form-urlencoded",
      "params": [
        {
          "name": "login",
          "value": ""
        }
      ]
    }
}
```

### PR DESCRIPTION
**Pull request by legander**
This PR fixes #28

### CODE REVIEW

File: example/params.har [hunk 1]
```diff
@@ -1,27 +1,42 @@
...
+              {
+                "name": "login",
+                "value": "admin"
+              },
+              {
+                "name": "password",
+                "value": ""
+              },
+              {
+                "name": "${userName}",
```
Comment (allansson) [inline example/params.har]: Shouldn't the param-name be hardcoded, and the variable be in the `value`-property?

Comment (legander) [reply example/params.har]: It is possible to use variables dynamically in urls, headers, queryParams etc. I added this to the example har just to see if the fix was still working for resolved cases of params.

Comment (legander) [reply example/params.har]: I can add the example for dynamic value as well 👍

**Comment by allansson** [comment]
Build error is caused by prettier not confirming to standardjs. I vote to remove standardjs in favor of prettier. :)

**Comment by legander** [comment]
Couldn't agree more with that! Can we do this in a separate PR tough? 😅

File: example/params.har (hunk 1 above)
Comment (allansson) [reply example/params.har]: Ah, then it was just the fact that the variable was called `userName` which threw me off.

**Comment by legander** [comment]
I'll fix the styling issues first

**Pull request merged**